README.md

```diff
@@ -201,7 +199,7 @@ steps:
 
 This will create a new Container App named `github-action-container-app-<github-run-id>-<github-run-attempt>` in a new
 resource group named `<container-app-name>-rg`. The Container App will be based off of an image that was built from
-the provided `appSourcePath` and pushed to the provided ACR instance. An access token will be generated to authenticate
+the provided `appSourcePath` and pushed to the provided Container Registry instance.If the Container Registry is ACR, an access token will be generated to authenticate

```

Comment:
We don't need to make any updates to this example description because we're explicitly using the `acrName` argument, so the description referring to ACR still applies